docs/building-for-ibm-dotcom.md

```diff
@@ -46,6 +46,20 @@ For English used in the United Kingdom, specify:
 the country identifier should be written in uppercase following usual 
 conventions.
 
+### Configuring available languages
```

Comment:
What's the thought process for putting this in this file vs. with the "current language documentation below". I'm wondering if they should be together, or at least cross-linked because of their high relationship towards one another.

Reply:
Added a link to this from the other README per the other comment.